src/app/clusters/general-commissioning-server/app_config_dependent_sources.cmake

```diff
@@ -18,6 +18,7 @@ TARGET_SOURCES(
   ${APP_TARGET}
   PRIVATE
     "${CLUSTER_DIR}/CodegenIntegration.cpp"
+    "${CLUSTER_DIR}/CodegenIntegration.h"
```

Comment:
No such file in this PR?

Reply:
This was probably missed before and things just compile because headers will work. File does exist, but I did not change it:

https://github.com/project-chip/connectedhomeip/blob/master/src/app/clusters/general-commissioning-server/CodegenIntegration.h